src/current/cockroachcloud/cloud-api.md

```diff
@@ -154,12 +163,15 @@ curl --request POST \
 {
   "name": "notorious-moose",
   "provider": "GCP",
+  "plan": "STANDARD",
   "spec": {
     "serverless": {
       "regions": [
         "us-central1"
       ],
-      "spendLimit": 0
+      "usage_limits": {
+        "provisioned_virtual_cpus": "2"
```

Comment:
As above, we should show 4 vCPUs in these examples, as the recommended minimum for Prod and the absolute minimum for Azure.